### DISCUSSION TITLE
install-sshd.ps1 dosen't work well.

### DISCUSSION THREAD

**Discussion by rea-jin** [PowerShell/Win32-OpenSSH#2023] [Q&A] [answered]
Hello, I have a question.

I downloaded OpenSSH-Win64 and ran `.\install-sshd.ps1`, but
 '`The term 'sc.exe' is not recognized as the name of a cmdlet, function, script file, or operable program.`'
But there is sc.exe in Windows/System32.

I looked it up and found out that this is old.
Then I downloaded openssh-portable and ran install-sshd.ps1 in the win32/openssh folder, but it said there was no sshd.exe.
So I copied the sshd.exe inside OpenSSH-Win64. Then when I run it  and I got an error 
'`The term 'sc.exe' is not recognized as the name of a cmdlet, function, script file, or operable program.`'
I ran a power shell with administrator privileges and also ran the command '`Set-ExecutionPolicy RemoteSigned`'.
What should I do?
I need some help.

OS:windows server 2019

**Comment by maertendMSFT** [comment, marked as answer]
Can you confirm that system32 is in your execution path?
You can run Get-Command sc.exe and it will a command not found error if it is not in the path.

**Reply by rea-jin** [reply]
Thank you for your reply.
'C:\Windows\System32' was set in the 'Path' of each of the user and system environment variables.
Therefore, I newly added '%SystemRoot%\System32' to the Path of the system environment variable.
Then Get-Command sc.exe found the execution path!
Thanks.
And.
I installed 'OpenSSH-Win64' and sshd worked fine.
"openssh-portable" is it not available on windows?

**Reply by tgauth** [reply]
The OpenSSH-Win64 zip/msi from https://github.com/PowerShell/Win32-OpenSSH/releases is the release of the Windows fork of OpenSSH-Portable. Hope that clarifies things!

**Reply by rea-jin** [reply]
I understand! Thank you very much.
